cc/simple_example.cc

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdio.h>
+#include <iostream>
+
+#include "cc/dual_net/factory.h"
+#include "cc/game.h"
+#include "cc/init.h"
+#include "cc/logging.h"
+#include "cc/mcts_player.h"
+#include "cc/platform/utils.h"
+#include "cc/random.h"
+#include "cc/zobrist.h"
+#include "gflags/gflags.h"
+
+// Inference flags.
+DEFINE_string(model, "",
+              "Path to a minigo model. The format of the model depends on the "
+              "inference engine. For engine=tf, the model should be a GraphDef "
+              "proto. For engine=lite, the model should be .tflite "
+              "flatbuffer.");
+DEFINE_int32(num_readouts, 100,
+             "Number of readouts to make during tree search for each move.");
+
+namespace minigo {
+namespace {
+
+void SimpleExample() {
```

Comment:
If this is the 'demonstration example' it might be worth giving a preamble or introduction indicating what the overall goal is and what ways the example might be extended (threading, auto-updating things, writing sgfs or examples, whatever).  I'm not sure how much the second part is necessary -- 'negative' comments are weird -- but it might be worth calling out how its different from its larger counterpart.

Reply:
done